lib/components/tree.component.ts

```diff
@@ -10,7 +10,7 @@ import { KEYS } from '../constants/keys';
 import * as _ from 'lodash';
 
 @Component({
-  selector: 'Tree',
+  selector: 'Tree, tree',
```

Comment:
How would you discern one selector's usage from another? If I try via the dom (el.nativeElement.localName), I get auto-lowercased values, and if I use reflect (http://stackoverflow.com/questions/37188216/accessing-selector-from-within-an-angular-2-component), I'll run into an AoT issue (https://github.com/angular/angular/issues/13495#issuecomment-267314384). 

If you've got any tips or tricks I'd be happy to hear them.